fireants/registration/moments.py

```diff
@@ -138,6 +138,63 @@ def find_best_detmat_3d(self, U_f, U_m, fixed_arrays, moving_arrays, com_f, com_
         ori = oris[best_idx, 0]
         return ori
 
+    def find_best_detmat_2d(self, U_f, U_m, fixed_arrays, moving_arrays, com_f, com_m, xy_f, xy_m):
+        '''
+        Find best determinant matrix for 2D
+        '''
+        # Define candidate orientation matrices (rotations / anti-rotations in 2D)
+        rot = [-np.eye(2) for _ in range(2)]
```

Comment:
the `rot` and `antirot` matrices in 3D are made to have determinants 1 and -1 respectively.

in the case of 2d, rot would contain the matrices [[1,0],[0,1]] and [[-1,0],[0,-1]]  and antirot would contain  [[1,0],[0,-1]] and [[-1,0],[0,1]]. This snippet needs to be written differently to handle it